src/components/calcite-block/calcite-block.stories.ts

```diff
@@ -0,0 +1,118 @@
+import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
+import { Attributes, createComponentHTML as create, darkBackground, parseReadme } from "../../../.storybook/utils";
+import blockReadme from "./readme.md";
+import sectionReadme from "../calcite-block-section/readme.md";
+
+export default {
+  title: "calcite-block",
+  decorators: [withKnobs],
+  parameters: {
+    notes: {
+      block: parseReadme(blockReadme),
+      section: parseReadme(sectionReadme)
+    },
+    backgrounds: darkBackground
+  }
+};
+
+const createBlockAttributes: () => Attributes = () => {
+  const group = "block";
+  const dirOptions = ["ltr", "rtl"];
```

Comment:
question: Maybe we could have constants we import for `dirOptions` and `themeOptions` since they will be the same throughout stories?

Reply:
💥